backend/pkg/api/data_access/vdb_blocks.go

```diff
@@ -382,7 +384,13 @@ func (d *DataAccessService) GetValidatorDashboardBlocks(ctx context.Context, das
 				Hash: t.Hash(hexutil.Encode(proposal.FeeRecipient)),
 			}
 			data[i].RewardRecipient = &rewardRecp
-			ensMapping[hexutil.Encode(proposal.FeeRecipient)] = ""
+			addressMapping[hexutil.Encode(proposal.FeeRecipient)] = nil
+			contractStatusRequests = append(contractStatusRequests, db.ContractInteractionAtRequest{
+				Address:  string(proposal.FeeRecipient),
```

Comment:
**issue:** I don't see this giving you the correct string formatting for `GetAddressContractInteractionsAt`.
It seems to request the address strings in the `fmt.Sprintf("%x", byteSlice)` format (so without the "0x") not as `string(byteSlice)`.
This affects all new uses of `GetAddressContractInteractionsAt` but please also see my comment in withdrawals.

Reply:
Ah you're right, good, ty